Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -14,15 +14,10 @@ public class ControllerPoseSynchronizer : InputSystemGlobalHandlerListener, IMix
     {
         #region IMixedRealityControllerPoseSynchronizer Implementation
 
-        [SerializeField]
-        [Tooltip("The handedness this controller should synchronize with.")]
-        private Handedness handedness = Handedness.Left;
```

Comment:
I believe this is removing a field in an asset, could this break people that depend on that somehow, or is it unused?

Reply:
Yeah, unless this is replaced somewhere in the same script, I think it should stick around. This class is intended to be used for more than just pointers.

Reply:
The existing bug here is that Handedness is both controlled via it's setter and via the Controller public property. This is a conflict because it doesn't make sense that Handedness is both the value I should synchronize with as well as my current handedness. 

For example, I could set handedness in inspector to left and then this is completely ignored by setting Controller property to the right one. Furthermore, I could then set Handedness to value Left again....even though I am assigned to the right controller.

Thus, either this becomes only a getter saying "hey I am currently this handedness" (which is current solution in this PR)
or
The Handedness property has a setter which is validated throughout the class (i.e you can't assign a Controller that has an invalid handedness)

@julenka  / @keveleigh thoughts on A vs B?

Reply:
Furthermore, I could set Handedness to Any and then Controller property sets it to Right. 
Originally, if I set Controller to null (which is done in the input system), then the handedness would still be Right...not Any which is what I had set in the inspector.

The lifetime of this property isn't properly managed so we need to go with either A or B above...or if you guys can provide a third option

Reply:
b) seems riskier, so it's okay to go with a) for now. @thalbern or @wiwei , regarding deprecating fields, do you have suggestions? Another suggestion is we could mark the property as obsolete for a release, and not use it internally anywhere. We could remove it in 2.4.

Reply:
I could mark it as obsolete...but it's a private field so I don't see how people could leverage it?

Reply:
Interesting...looking through, I thought this script used to do more as far as "set the handedness in the inspector, then wait for a controller to be detected with that handedness and synchronize". It doesn't appear to do anything like that anymore.
Of course, that's also probably a better use case for solvers than this. I think I'm less concerned about this removal now, since...it doesn't do anything? At least in the base class.

Reply:
I just deprecated for now, but it at bottom of class